packages/framework/src/Component/Image/Processing/ImageProcessor.php

```diff
@@ -162,17 +160,4 @@ public function getSupportedImageExtensions()
     {
         return $this->supportedImageExtensions;
     }
-
-    /**
-     * @param string $filepath
-     * @param string $newFilepath
-     */
-    protected function removeFileIfRenamed($filepath, $newFilepath)
```

Comment:
Removed method is not mentioned in upgrade notes and probably should be.

Reply:
I was thinking about it as well, however, I decided not to mention it as this is a protected method. So the upgrade note would look like "If you are extending ImageProcessor class and if you are using its method removeFileIfRenamed(), you need to implement it yourself now as the method was removed"...seems like too many ifs in one instruction to me...what do you think?

Reply:
Since it is a little edge-case to me (method was private a month ago), I would simplify it just to "if you're using protected method ImageProcessor::removeFileIfRenamed in your code, remove the usage due to the method was removed and the code was moved to `convertToShopFormatAndGetNewFilename`".

Reply:
but I can live without this upgrade note (private month ago)

Reply:
ok, I like your formulation so I added it there :slightly_smiling_face: